app/ui/styles/components/spinner.css

```diff
@@ -13,6 +13,11 @@
   animation: rotate 5s linear infinite;
 }
 
+.spinner .path,
+.spinner .bg {
+  stroke: currentColor;
```

Comment:
When a button is disabled this makes it look dull because the text colour of the button becomes something like `text-accent-disabled`. Need to investigate.